src/PersonObjects/Player/PlayerObjectGeneralMethods.tsx

```diff
@@ -1496,13 +1496,14 @@ export function startCrime(
   chaExp: number,
   money: number,
   time: number,
+  focus: true,
```

Comment:
focus:true in the type definitions of the params for startCrime... focus should be type defined as a boolean and set to a default value of true.